include/setup.php

```diff
@@ -570,7 +570,6 @@ function checkSendingAuthHeader($rep = false) {
 	}
 	$loggedin = $auth->hasUsername();
 	/*if (!$loggedin) {
-		header('WWW-Authenticate: Basic realm="'.str_replace('"', '\"', $auth->getBasicRealm()).'"');
 		header('HTTP/1.x 401 Unauthorized', true, 401);
 	} else {*/
```

Comment:
Consider remove commented code, is versioned anyway.

Reply:
Makes sense!